scripts/pylint_extensions_test.py

```diff
@@ -1143,3 +1143,143 @@ def func3():
 
         with checker_test_object.assertNoMessages():
             temp_file.close()
+
+
+class SingleNewlineAboveArgsCheckerTests(unittest.TestCase):
```

Comment:
Can you also add two more tests:
- Error also triggers if there are no spaces above returns (You'll have to update the error messages to reflect that)
- No error if there are no Args or Return section.

Reply:
Done.